REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,62 @@
+### Remaining public aliases
+
+annotation_reader
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
+security.acl.provider
+security.encoder_factory
+security.password_encoder
+session.handler
+session.storage
+templating.loader
+translator
+
+cache.app_clearer <- for the cache:pool:clear command to clear app pools
+$commandId <- for non-lazy commands which are get() at runtime
+router <- for the base controller
+templating <- for the base controller
+
+### Remaining public services
+
+file_locator
+filesystem
+kernel
+property_accessor
+property_info
+security.acl.dbal.schema
+security.authentication_utils
+test.client
+translation.extractor
+translation.loader
+translation.reader
+translation.writer
```

Comment:
@Nyholm do you think we should plan making private these translation.*/translator services in 4.0?

Reply:
`translator` must stay public. Otherwise, people using ContainerAware classes cannot use the translator layer anymore

Reply:
note added. any comment I could add on the remaining aliases/services?

Reply:
IMHO, all theses services (above) could be private. except the `security.authentication_utils` (for use in ContainerAware classes)